drivers/amazonec2/amazonec2.go

```diff
@@ -821,6 +818,18 @@ func (d *Driver) innerCreate() error {
 		return err
 	}
 
+	if d.RequestSpotInstance {
```

Comment:
Where do you wait for the spot instance to be running before applying tags? It looks like it checks once and if it doesn't exist yet, doesn't apply tags